### ISSUE TITLE
connecting to wallet waiting !!!!!

### ISSUE THREAD

**Issue by teddy840** [Chia-Network/chia-blockchain#3851]
help !!
![Capture](https://user-images.githubusercontent.com/83728216/117198188-5e8db180-adf1-11eb-82a1-6796d9969263.PNG)

**Comment by BotaniQue** [comment]
Go to Task Manager and kill all hanging processes:
chia.exe
daemon.exe
start_farmer.exe
start_full_node.exe
start_harvester.exe
start_wallet.exe

Then try again.

**Comment by teddy840** [comment]
tnx you
the problem is after i close the chia all the plotting cancel

**Comment by gregordr** [comment]
Same issue here, but I don't even have all the tasks except for chia.exe

**Comment by meikmenzel** [comment]
Yes, same here. Also restarted my pc (win 10). But still: "Connecting to wallet"...

**Comment by Kolerazie** [comment]
Same issue :(

**Comment by johnson49235** [comment]
Force Restart causes the same problem.  Will the plotting continue if i do not restart?

**Comment by hdsreggae** [comment]
me pasa lo mismo despues de un cierre inesperado , por que entro en modo de ahorro de energia .. luego no pude conectarme mas :(

**Comment by johnson49235** [comment]
No sé si se van a acabar mis parcelas. Voy a pasar de wifi a ethernet.
He oído que es posible que se pierda bloques de wifi.

**Comment by dalvarez94** [comment]
sigo asi desde ayer

**Comment by abbail** [comment]
Same issue here, the developer tools is stuck in a loop:
```
...
Already connected, not reconnecting.
0
Already connected, not reconnecting.
0
...
```
None of the solutions above worked.  I even rebooted and reinstalled the client.  I'm on 1.1.4 but I also tried 1.1.3 and 1.1.2 and got the same results.

**Comment by oliman** [comment]
I am having the same problem, didn't notice it never connected after upgrading my full node. This has kept me offline all day. Can someone suggest a fix? I have restarted, uninstalled/re-installed still always stuck at connecting to wallet

**Comment by trinitiwowka** [comment]
Та же проблема. не загружается. macbook pro 2011. на всех остальных устройствах запускается

**Comment by Aspisen** [comment]
It's literally the sync file inside the wallet in your Chia files that are blocking you from logging in.

If you go to your directory, mine is at C:\Users\Username\.chia\mainnet\wallet\db

And remove the largest file you will be allowed to log in again, but all your sync is removed.
You can copy the file out and keep it somewhere else for now if they come up with a fix later on.

I'm re-syncing right now and plotting. No reinstall needed.

**Comment by Kolerazie** [comment]
Doesn't work for me! I removed the blockchain_wallet_v1_mainnet_28421 ------. Sqlite from the directory c: \ users \ username \ .chia \ mainnet \ wallet \ db \.

or do you mean the c: \ users \ username \ .chia \ mainnet \ db \ sqlite file?

I'm back to 1.1.3 afterwards, without success!

**Comment by Aspisen** [comment]
Yes the c: \ users \ username \ .chia \ mainnet \ db \ sqlite file is the one I mean :)

**Comment by migelrestrepo** [comment]
It worked for me on mac. I did the following.
Run the following command in terminal to show hidden files
defaults write com.apple.Finder AppleShowAllFiles -bool YES
Then this command
killall Finder
Then I have gone to the path User /.chia/mainnet/Wallet/db and I have eliminated the largest file. 
With that it has worked for me again.

**Comment by Kolerazie** [comment]
so wallet / db after all? no matter which one I delete, it doesn't help. since you stayed on 1.1.4?

win10

**Comment by savire** [comment]
Too many bugs and they don't seem to bother at all. After waiting 24 hours on relatively high CPU and RAM usage along with I/O too. Its no different than BitCoin in the end. 

We can't really use our rig for other tasks once its being used for plotting. Not to mention its suddenly stopped the process if something wrong happen.

**Comment by msn1983aa** [comment]
ploted 95% when this happen，so angry！waste 14hrs！

**Comment by Havoc738** [comment]
Same issue here.

**Comment by Khunmatthew** [comment]
> ploted 95% when this happen，so angry！waste 14hrs！

me too, my phots wasted!

**Comment by abbail** [comment]
I was able to solve this for myself.  I downloaded a sqllite viewer and opened the "C:\Users\%USERNAME%\.chia\mainnet\db\blockchain_walleter_v1_mainnet_#####.sqlite".  In the block_records table I scrolled to the end and found the first record with an weight of Inf (infinity) and deleted it and all records after it.  I then closed and saved the database.  I was then able to start the client.  Remember to kill all associated running processes.  Hope this helps someone else.
